firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsReportDataCapture.java

```diff
@@ -137,12 +137,20 @@ public Event captureEventData(
   }
 
   private CrashlyticsReport.Session.Application populateSessionApplicationData() {
-    return CrashlyticsReport.Session.Application.builder()
-        .setIdentifier(idManager.getAppIdentifier())
-        .setVersion(appData.versionCode)
-        .setDisplayVersion(appData.versionName)
-        .setInstallationUuid(idManager.getCrashlyticsInstallId())
-        .build();
+    CrashlyticsReport.Session.Application.Builder builder =
```

Comment:
For consistency, can we make this `final`?

Reply:
Done